### DISCUSSION TITLE
Transition

### DISCUSSION THREAD

**Discussion by michael-vasyliv** [tailwindlabs/headlessui#730] [Ideas]
Hello! Let's say I want to use `Transition` for animation from state `opacity-50` to state `opacity-75` I don't want to unmount or hide a component I want have it shown all time (but it's impossible since this [line](https://github.com/tailwindlabs/headlessui/blob/5af3bd4b7164a0205ac70c179d8e9651afbb0a72/packages/%40headlessui-react/src/components/transitions/transition.tsx#L212))

**Comment by Ding-Fan** [comment]
Same issue, I have to manually write "show elements beforeEnter and afterLeave". and the layout is going mad while switching elements.
I'm trying to implement a flipping card. It could simply be done with toggle class. Maybe this is not the use case of headless's Transition.
